packages/backend-for-frontend/src/routers/clientRouter.ts

```diff
@@ -345,7 +346,30 @@ const clientRouter = (
         );
         return res.status(errorRes.status).send(errorRes);
       }
+    })
+
+    .delete("/clients/:clientId/admin/:adminId", async (req, res) => {
+      assertAdminClientFeatureEnabled(res);
```

Comment:
How this should works? 
If assertion fails `res.status` was invoked, but without return the execution continues, ad status will be overwritten with 204 if  await clientService.removeClientAdmin successed otherwise return an internal server error or a specific error from clientService.